OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -1834,6 +1834,8 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
                 List<DreamValue> values;
                 if (value.TryGetValueAsDreamList(out DreamList list)) {
                     values = list.GetValues();
+                    if (values.Count == 0)
+                        throw new Exception("pick() from empty list");
```

Comment:
This should be moved outside here and above the `picked = ...` line so that `pick(arglist(list()))` is caught as well.

Reply:
Yeah, that's right, thanks. Fixed